src/selectors/balances.js

```diff
@@ -135,9 +135,9 @@ const liquidityPoolsBalanceListSelector = createSelector(
       stakedAmount,
     } = getPoolStats(pool, liquidityPoolsState) ?? {};
 
-    if ([currentPrice, userLiquidityTokenBalance, stakedAmount].includes(undefined)) return null;
+    if (currentPrice === undefined) return null;
     return {
-      balance: (userLiquidityTokenBalance + stakedAmount) * currentPrice,
+      balance: (userLiquidityTokenBalance.toNumber() + stakedAmount.toNumber()) * currentPrice,
```

Comment:
So are we sure `userLiquidityTokenBalance` and `stakedAmount` can never be `undefined`? Also, shouldn't we use BN to calculate?

Reply:
Re null checks: I're right, these could happen for `null` `poolStats` getting coalesced to `{}`. I've re-added them

Reply:
Re: BN calculation, these are rough calculation for fiat display, so I left them in `number` calculation.